src/components/EventsTimeline/TimelineEventItem.tsx

```diff
@@ -44,10 +52,25 @@ const TimelineEventItem: React.FC<Props> = ({ fields, caseEvents, title = "", da
               >
               <EventWrapper fields={ fields } caseEvent={ caseEvent } isNested={ true } />
             </Timeline> :
-            <EventWrapper fields={ fields } caseEvent={ caseEvent } />
+            <>
+              <EventWrapper fields={ fields } caseEvent={ caseEvent } />
+              { items.length > 0 &&
+                <Dl>
+                  { items.map(({ label, value, value_verbose }, index) => { //value_verbose is the humanreadable variant of value
+                    if(value === false) return null //hide unchecked checkboxes
+                    return (
+                      <div key={ `${ label }_${ index }` }>
+                        <dt>{ label }</dt>
+                        <dd>{ typeof value === "string" && value_verbose }</dd>
```

Comment:
This will be empty when value is not a string?

Reply:
And checks for `value`, but prints `value_verbose`. Is that correct? Maybe do: `typeof value_verbose === "string" ? value_verbose : "-"`

Reply:
Yes. It's valid html